go.mod

```
@@ -256,7 +256,7 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.27.0-sdk-v0.46.16
+	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.27.0-sdk-v0.46.16.0.20250327122859-1f9452635467
```

Ideally we cut a release of celestiaorg/cosmos-sdk so that this can use an official release

---

Created https://github.com/celestiaorg/cosmos-sdk/releases/tag/v1.28.0-sdk-v0.46.16
```suggestion
	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.28.0-sdk-v0.46.16
```

---

plus a `go mod tidy`